docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -3029,6 +3029,79 @@ The price bump percentage to replace an existing transaction. The default is 10.
 The maximum period, in hours, to hold pending transactions in the transaction pool. The default is
 13.
 
+### Xdns-enabled
+
+=== "Syntax"
+
+    ```bash
+    --Xdns-enabled=[<true|false>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --Xdns-enabled=true
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    BESU_XDNS_ENABLED=true
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    Xdns-enabled=true
+    ```
+
+Enables DNS support. The default is `false`.
+
+!!! important
+
+    Use domain names in private networks only because public networks require using IP addresses.
```

Comment:
Suggest updating to: `Use DNS with trusted DNS provider in private networks because of limitations where IP address can change eg Kubernetes pods`

Reply:
Done